extensions/transform/org.eclipse.smarthome.transform.regex/README.md

```diff
@@ -50,7 +50,7 @@ The regex pattern is is defined as follows
 *  `()` capture group match 
     * `\d*` match a digit (equal to [0-9]), zero and unlimited times, the backslash has to be escaped see [string vs plain](#Differences-to-plain-Regex)
     * `.` match the dot literally
-    * `\d*` match a digit (equal to [0-9]), zero and unlimited times, the backslash has to be escaped see [string vs plain](#Differences-to-plain-Regex)
+    * `\w*` match a word (equal to [a-Z]), zero and unlimited times, the backslash has to be escaped see [string vs plain](#Differences-to-plain-Regex)
```

Comment:
`\w` is actually defined as `[a-zA-Z_0-9]`.

Reply:
> \w is actually defined as [a-zA-Z_0-9]
O yeah, my bad.

Next time I will use the right way to pull some changes.
But this was the way the OpenHAB documents told me to follow.

Reply:
Please change the definition in your branch accordingly and push your changes again. It will directly update the PR here. Thanks.